server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -123,7 +123,7 @@ export class BasicRestWrapper extends RestWrapper {
         const options = { ...requestConfig };
         options.headers = this.generateHeaders(
             options.headers,
-            (this.getCorrelationId && this.getCorrelationId()) || uuid());
+            (this.getCorrelationId?.()) || uuid());
```

Comment:
Nits:

```suggestion
            this.getCorrelationId?.() ?? uuid(),
        );
```

Reply:
Haha, the nullish coalescing change is in a separate PR so I left it here.